plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go

```diff
@@ -77,6 +79,9 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// check and default
```

Comment:
?

Reply:
I think this was more of a top-level 'what are we doing here' type comment, but in context it is pretty useless. I'll remove it, but I think we should try and get a group together to review the default service plan admission controller.